More grammar fixes - Heliarch and Lunarium intros (#9633)
endless-sky/endless-sky: data/coalition/heliarch intro.txt

			branch license
				has "joined the heliarchs"
			label waited

> 			`	A few more minutes pass, and you hear some more of the heated discussion from within. After a while it stops and the door opens, with Aulori on the other side, alone. "Arrived you have, I see, Captain. Forgive me for the wait you must. Just going over the findings of the expedition, some more consuls and I were."`

Reviewer: ```suggestion
			`	A few more minutes pass, and you hear some more of the heated discussion from within. After a while, it stops, and the door opens with Aulori on the other side, alone. "Arrived you have, I see, Captain. Forgive me for the wait, you must. Just going over the findings of the expedition, some more consuls and I were."`
```

			label beckon
			`	He beckons you to the middle of the room, still lit up only by the many monitors used for the video calls with other consuls. He moves to a small desk near the wall with the monitors, and picks up <payment> in chips, handing them to you. "Find what they could with what time they had, the expedition did. Only so much time to experiment there was, when a single ship - a warship - the fleet had. Mistake me not, greatly pleased with the preliminary results, I so far have been, but frustrating it is that limited we are by such time and resource constraints." He sighs. "Like I would, for there to be a larger, more substantial expedition to that ring. But stand as it does, seem it does that wait I must for the council's approval, before taking any further action."`
			`	He moves to pick up a small box on the other side of the room, sitting atop a table surrounded by boards with scribbled notes and mock-up schematics. He brings the box to you and opens it, revealing inside a very small piece of something clearly metallic in nature, but with some odd, unknown property to it you can't identify. "One billion."`
			choice
				`	"What?"`
				`	"Is this that expensive?"`

> 			`	"Over the course of the War of Independence, the amount of kilotons of fissile material used in projectiles that bombarded this ring. One billion. For this ring alone. They left not one dent. Not one." He picks up the piece. "One of the smaller shards of the shattered ring that recover, the expedition could." He gazes at it for a few seconds, before putting it back in the box and returning it to the table. "In my position, to learn of the Quarg, learn as a Quarg, I have had to. Think like them, I must, if to attempt to breach the many blockades they have placed to impede others from learning from their technology, I am. Outnumbered by them, we are. Inferior we remain in our technology. If a means to close the gap, I do not find, then perhaps learning what monstrous weapon could break their rings, learning how to create such a weapon, the only hope for deterring a Quarg offensive is."`

Reviewer: ```suggestion
			`	"Over the course of the War of Independence, the amount of kilotons of fissile material used in projectiles that bombarded this ring. One billion. For this ring alone. Not one dent, they left. Not one." He picks up the piece. "One of the smaller shards of the shattered ring that recover, the expedition could." He gazes at it for a few seconds, before putting it back in the box and returning it to the table. "In my position, to learn of the Quarg, learn as a Quarg, I have had to. Think like them, I must, if to attempt to breach the many blockades they have placed to impede others from learning from their technology, I am. Outnumbered by them, we are. Inferior in our technology, we remain. If a means to close the gap, I do not find, then perhaps learning what monstrous weapon could break their rings, learning how to create such a weapon, the only hope for deterring a Quarg offensive is."`
```
Soft suggestions. I'm not well versed in Coalition translator grammar.

			choice
				`	"<first> <last>. Pleasure to meet you."`
				`	"What job did you have for me?"`
			`	"In need of one such as you, we are. Received anonymous reports, we have, from loyal patriots and colleagues alike. On <stopovers>, a Saryd world, brewing, terrible activity may be. Rising in activity there, outlaw groups are, we fear," he says. "Your destination, that is."`
			`	A couple small boxes are brought inside and set on the table. Upon opening them, Plortub shows you they contain several small drones - drones very similar to those employed by the hull repair modules used in Coalition ships.`

> 			`	"A union of technologies, it is. Of the scanning module, the incredible surveillance capabilities are. Of the repair drones, the versatile mobility and durable shells are." He explains to you that you're to head to <planet stopovers> to pick up <cargo> produced on the planet, but as you're coming in for a landing, you are to jettison the crates with the drones so that they may act as spybots. "Be inconspicuous, you must. Appear to be a simple, regular job, this must. Return to <origin>, you must, after completed this task, you have. Readily available then, your payment will be. Suffice, <payment> will, surely."`

Reviewer 2: I see.

	on offer
		conversation
			`You're stopped by one of the many armed Heliarchs in the spaceport, and are asked to follow him into the restricted sections. You comply and are led through the corridors until you're brought before Consul Plortub, in a smaller room than last time. Three Kimek Heliarchs are with him.`

> 			`	"Ask for your assistance, once again, I must, Captain <last>," he begins. "Long been a difficult world for the Heliarchy, <destination> has. Admit I must, that known and, to my shame, ignored, lawless behavior there has been. Your continued support, you'll once more provide we hope, Captain."`

Reviewer: ```suggestion
			`	"Ask for your assistance, once again, I must, Captain <last>," he begins. "Long been a difficult world for the Heliarchy, <destination> has. That known and, to my shame, ignored, lawless behavior there, admit I must. Your continued support, you'll once more provide we hope, Captain."`
```
Soft suggestion

			label license
			`	He stares at you for a few seconds, almost like a statue. "Afraid I am that orders, these are. Received them directly from the Ring of Friendship, from Consul Iekie, I have. A great asset for this task, you would be, Captain, so insist upon your involvement, I must."`
			label plan

> 			`	He introduces you to the three Kimek, and goes to explain the plan. "Been granted access to our civilian ships, you have. Three new Kimek Spires, you have very recently bought. Carrying many passengers to <destination>, in them you are. A very lucrative job, it is." He pauses, and gives the Kimek permission to head out. "Pose as civilians, working in your ships, they will. Armed and ready to investigate <planet> thoroughly, the soldiers in those Spires are. Instructed to follow your ship there, these three were."`

Reviewer: ```suggestion
			`	He introduces you to the three Kimek and proceeds to explain the plan. "Been granted access to our civilian ships, you have. Three new Kimek Spires, you have very recently bought. Carrying many passengers to <destination>, in them you are. A very lucrative job, it is." He pauses and gives the Kimek permission to head out. "Pose as civilians, working in your ships, they will. Armed and ready to investigate <planet> thoroughly, the soldiers in those Spires are. Instructed to follow your ship there, these three were."`
```

Reviewer: You can keep the comma if you reword it like this:
`He pauses for a moment, and he gives the Kimek permission to head out.`

You can't have a comma in this context unless there's an independent clause on the other end. Adding the subject `he` creates an independent clause, and I think the text flows better if you also add `for a moment` (though that last bit is not grammatically required).

		payment 495000
		conversation
			`You land in the station to find that <payment> have been transferred to you. Along with the credits comes a transmission from Plortub. "For my absence, forgive me you must, Captain. Hit Fourth Shadow again, criminals have, so organizing some ships from within the Ring of Power, I am."`

> 			`	He goes on to explain that the attack was quite a large one, and much of the fleet stationed here was sent there to help contain it. What remained is helping with the <origin> operation, so the station is fairly devoid of a military presence at the moment. "A troubled week, this will be, but bother you with that, I will not. Been very useful to us your support has, Captain."`

Reviewer: ```suggestion
			`	He goes on to explain that the attack was quite a large one, and much of the fleet stationed here was sent to help contain it. What remains of the fleet is helping with the <origin> operation, so the station is fairly devoid of a military presence at the moment. "A troubled week, this will be, but bother you with that, I will not. Been very useful to us your support has, Captain."`
```

or

```suggestion
			`	He goes on to explain that the attack was quite a large one, and much of the fleet stationed here was sent to help contain it. The remaining ships are helping with the <origin> operation, so the station is fairly devoid of a military presence at the moment. "A troubled week, this will be, but bother you with that, I will not. Been very useful to us your support has, Captain."`
```

		conversation
			`Though you never managed to find out if there were any hostages on board, the Heliarchs stationed at <planet> still herald you as a hero for "detaining the bandits". They immediately set to repair your ship. You're still on alert for a few hours in case any more ships come through, but nothing appears to be out of the ordinary. Near the end of the day, the station returns to normal as well; communications and other functions are restored.`
			`	As soon as the commlinks are back, the Heliarchs contact Consul Plortub and inform him of what happened. Shortly after, <payment> are added to your account, and you're put on a call with the consul.`

> 			`	"Humiliating, it would be, had under such circumstances, the criminal ships escaped. Done well, you have, Captain." He discusses with you the outfits that the ships were using and is alarmed to learn that they had not only Heliarch equipment, but had also acquired human weapons. He lets out something you can't quite get, resembling a gasp or a roar; you only know that your ears don't want to hear it again.`

Reviewer: ```suggestion
			`	"Humiliating, it would be, had under such circumstances, the criminal ships escaped. Done well, you have, Captain." He discusses with you the outfits that the ships were using and is alarmed to learn that they had not only Heliarch equipment, but they had acquired human weapons as well. He lets out something you can't quite get, resembling a gasp or a roar; you only know that your ears don't want to hear it again.`
```

		not "assisting lunarium"
	on offer
		conversation

> 			`Once again, you managed to avoid the Hunters while landing on <origin>, and a local Heliarch agent informs Magister Lotuora of your success. "Excellent work, Captain <last>," she says when she contacts you. "Now, talked with some contacts I have, and one more drill for you to help with, I have arranged. Return to <planet> at your earliest convenience, you may." She signs off, but calls back seconds after. "Ah, forgive me you must, Captain. Forgotten something, I had. Hear of the recent drills, some colleagues did, and readied their own Hunters to participate, they now have. Avoid them on your return to <planet>, as per usual, you should."`

Reviewer: ```suggestion
			`Once again, you managed to avoid the Hunters while landing on <origin>, and a local Heliarch agent informs Magister Lotuora of your success. "Excellent work, Captain <last>," she says when she contacts you. "Now, talked with some contacts I have, and one more drill for you to help with, I have arranged. Return to <planet> at your earliest convenience, you may." She signs off but calls back seconds after. "Ah, forgive me you must, Captain. Forgotten something, I had. Hear of the recent drills, some colleagues did, and readied their own Hunters to participate, they now have. Avoid them on your return to <planet>, as per usual, you should."`
```

Reviewer: Piggybacking on my previous comment about independent clauses – you can keep the comma if you want, but then you have to add a subject on the other side of the comma.

`She signs off, but then she calls back again seconds later` for example.

Reviewer: I understand what you're going for. That mental narration is a common source of comma splices; I have to self-correct myself from doing it all the time.

If you want the same effect, you can use an ellipsis (`...`), which would be grammatically correct and still evoke a pause.

```suggestion
			`Once again, you managed to avoid the Hunters while landing on <origin>, and a local Heliarch agent informs Magister Lotuora of your success. "Excellent work, Captain <last>," she says when she contacts you. "Now, talked with some contacts I have, and one more drill for you to help with, I have arranged. Return to <planet> at your earliest convenience, you may." She signs off... and calls back seconds after. "Ah, forgive me you must, Captain. Forgotten something, I had. Hear of the recent drills, some colleagues did, and readied their own Hunters to participate, they now have. Avoid them on your return to <planet>, as per usual, you should."`
```
